<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRenderScene</class>
 <widget class="QWidget" name="QmitkRenderScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rendering Scene</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="btnStartRendering">
     <property name="text">
      <string>Start rendering</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="chkScroll">
     <property name="text">
      <string>Scroll each frame</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="btnStopRendering">
     <property name="text">
      <string>Stop rendering</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="chkModifyData">
     <property name="text">
      <string>Modify data each frame</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QFrame" name="renderWindowContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnLoad">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Load data...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Update rate (desired) [Hz]</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Update rate (achieved) [Hz]</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="spnDesiredRenderRate">
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="achievedRenderingDuration">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Rendering duration [ms]</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="achievedRenderingRate">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Load data presets</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnData1">
       <property name="text">
        <string>2D image w/ measurements</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnData2">
       <property name="text">
        <string>2D image as plane in 3D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnData3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="chkWriteSimpleText">
     <property name="text">
      <string>write text (PlanarFigures)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="chkBinaryContours">
     <property name="text">
      <string>contours (binary image)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="label"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
